src/Logging/Logging.EventLog/src/EventLogLoggerProvider.cs

```diff
@@ -49,6 +49,7 @@ public ILogger CreateLogger(string name)
 
         public void Dispose()
         {
+            _settings.Dispose();
```

Comment:
Instead of making settings disposable can't we access the internal EventLog property here directly and dispose it?

Reply:
That was my first thought, but I change it because I thought every class should dispose its resources in case in future they have more resource to dispose.

Reply:
I rather not make a settings type publicly disposable since the IEventLog being stored there is an implementation detail.